forest/src/daemon.rs

```diff
@@ -125,12 +125,20 @@ pub(super) async fn start(config: Config) {
 
     // Initialize database (RocksDb will be default if both features enabled)
     #[cfg(all(feature = "sled", not(feature = "rocksdb")))]
-    let db = db::sled::SledDb::open(format!("{}/{}", config.data_dir, "/sled"))
-        .expect("Opening SledDB must succeed");
+    let db = db::sled::SledDb::open(format!(
+        "{}/{}/{}",
+        config.data_dir, &config.chain.name, "/sled"
+    ))
+    .expect("Opening SledDB must succeed");
 
     #[cfg(feature = "rocksdb")]
-    let db = db::rocks::RocksDb::open(PathBuf::from(&config.data_dir).join("db"), &config.rocks_db)
-        .expect("Opening RocksDB must succeed");
+    let db = db::rocks::RocksDb::open(
+        PathBuf::from(&config.data_dir)
+            .join(&config.chain.name)
+            .join("db"),
+        &config.rocks_db,
```

Comment:
This is a breaking change. If we do not plan to offer a db migration, we should at least highlight it in the release notes.

Reply:
Agree, we should definitely document this.